v2/internal/testcommon/resource_namer.go

```diff
@@ -132,7 +146,7 @@ func (n ResourceNamer) GeneratePasswordOfLength(length int) string {
 	// This pass + <content> + pass pattern is to make it so that matchers can reliably find and prune
```

Comment:
I thought about suggesting we also expand the collection of runes to choose from here (in the spirit of making things more secure) but realized that's likely to cause problems as at least some applications we use this for (SQL, etc) may be picky about the charset of passwords...

For now I think this is probably OK.